model/src/main/proto/exploration.proto

```diff
@@ -420,3 +420,32 @@ enum CheckpointState {
   // Progress made in the exploration is not saved.
   CHECKPOINT_UNSAVED = 3;
 }
+
+// Corresponds to a item selection answer that user has selected.
+message ItemSelectionAnswerState {
+  repeated int32 selected_indexes = 1;
+}
+
+// Corresponds to a raw representation of the current answer entered by the user
+// which is used to retain state on configuration changes
+message UserAnswerState {
+  AnswerErrorCategory answer_error_category = 1;
+  oneof answer_input_type{
+    ItemSelectionAnswerState item_selection = 2;
```

Comment:
Don't forget to add docs for all the fields.

Reply:
Okay.